examples/bri-3/src/bri/zeroKnowledgeProof/api/anchorHash.controller.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { BadRequestException } from '@nestjs/common';
```

Comment:
should not all tests have a test in the name and be in the appropriate folder? I am a bit confused. Maybe just noticed it now, or I am missing something.

Reply:
see also naming

Reply:
@Therecanbeonlyone1969 the same was suggested by me some time ago, and currently we are following the specification by NestJs, suggesting to have all the test files close to the file under test. Additionally, NestJs suggests to have `.spec` in the test file name

Reply:
per recommendation from NestJs and as a general rule of thumb when it comes to TypeScript testing, you want unit tests to be close to the thing they are testing. .spec is a common naming pattern for test files.